Automattic/wp-calypso
# Remove Menus code

This is cleanup from #12967 , where the Menus link was replaced by links to the Customizer.

To test:
* Restart Calypso
* Make sure that it still works as before.
* Do some `grep`ping for `menus` to make sure we've found all occurrences.

Last open question:
* ~Do we also want to remove menus related stuff from `wpcom-undocumented`?~ Removed.
